browser/app/js/browser/Login.js

```diff
@@ -46,7 +46,7 @@ export class Login extends React.Component {
         password: document.getElementById("secretKey").value
       })
       .then(res => {
-        history.push(minioBrowserPrefix)
+        history.push("/")
```

Comment:
Should we remove the `import { minioBrowserPrefix } from "../constants"` line on top of this file as we don't use it here anymore?

Reply:
yes, will do